hazelcast-spring/src/main/java/com/hazelcast/spring/AbstractHazelcastBeanDefinitionParser.java

```diff
@@ -398,7 +398,7 @@ protected BeanDefinition getEvictionConfig(final Node node) {
             }
 
             try {
-                EvictionConfigHelper.checkEvictionConfig(evictionPolicyValue, comparatorClassNameValue, comparatorBean);
+                checkEvictionConfig(evictionPolicyValue, comparatorClassNameValue, comparatorBean, false);
```

Comment:
This method has now lost a bit of a readability. The false argument makes no sense when reading the invocation? 

Can't you add an method checkNearCacheEvictionConfig method which puts in false for this argument?